src/containers/community/CommunityEditor.js

```diff
@@ -36,6 +37,7 @@ const categories = {
   let validating = some(communityValidation.pending)
   let { community, errors } = communityEditor
   let saving = pending[CREATE_COMMUNITY]
+  let uploadingImage = !!pending[UPLOAD_IMAGE]
```

Comment:
I did this in the most verbose way, by adding a new prop. Another way would be to roll it in to `saving`, maybe rename `saving`
ie
`let pending = pending[CREATE_COMMUNITY] || pending[UPLOAD_IMAGE]`
figured it's better to have it more articulated.